src/style.css

```diff
@@ -1 +1,100 @@
-/* Styles go here */
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
do not reset margins and paddings for every element - it could affect performance